### PR TITLE
Bump nodeJS version to 20 (LTS)

### PR DESCRIPTION
**Pull request by chuttam**
v16 is deprecated and several packages no longer build against it (core dependency installation failures).

### CODE REVIEW

**Comment by chuttam** [comment]
Tagging @wgrrrr for review.

Note: Do not merge when approved, since that will auto-push to dockerhub. Still need to test extensively before merging.

**Review by wgrrrr — APPROVED** [review]

**Comment by wgrrrr** [comment]
This requires a bunch of infrastructure work to fully deploy to production. All management instances will need to have this installed. This means:
- update the `er-infrastructure` repo to deploy this version
- rebuild the management AMI with this new version
- replace all management instances using this new AMI

**Pull request merged**
